patterns/1-initial/creating_an_innersource_strategy.md

```diff
@@ -0,0 +1,78 @@
+## Title
+
+Creating an InnerSource Strategy
+
+## Patlet / Summary
+
+Sometimes, it is difficult to convince people of the relevance of InnerSource for your organization and/or to get support from management. Creating an InnerSource strategy, that connects your InnerSource approach and activities to the goals and the overall strategy of your organization, can help in this regard.
+
+## Problem
+
+Despite an organization has an InnerSource program, it is challenging to establish InnerSource in that organization. Missing management support and low awareness in some or many development teams are among the reasons. Potentially, the InnerSource program itself is missing clear goals and/or approaches how to achieve those.
+
+## Context
+
+InnerSource is not a means to an end. It must deliver clear benefits to your organization and support its goals. If InnerSource and its benefits is mainly explained and positioned in a rather abstract way, it might be difficult to convince management and a critical mass of the development teams. People might say/think "InnerSource sounds great, but it is not relevant for our organization".
```

Comment:
@misappi , did you intend to say here that InnerSource in not an _end_ in itself, it is a mean to an end?

Reply:
"... not an end in itself ..." -> corrected